crypto/x509/by_store.c

```diff
@@ -152,13 +199,13 @@ static int by_store(X509_LOOKUP *ctx, X509_LOOKUP_TYPE type,
                     const OSSL_STORE_SEARCH *criterion, X509_OBJECT *ret,
                     OSSL_LIB_CTX *libctx, const char *propq)
 {
-    STACK_OF(OPENSSL_STRING) *uris = X509_LOOKUP_get_method_data(ctx);
+    STACK_OF(CACHED_STORE) *uris = X509_LOOKUP_get_method_data(ctx);
```

Comment:
This variable name no longer seems appropriate

Reply:
Good catch, thanks